components/topic-overview/TopicOverviewNav.tsx

```diff
@@ -34,12 +34,12 @@ export const TopicOverviewNav = ({selected, setSelected}:Props) => {
 
     return (
         <>
-            <Container style={{maxWidth: 'none'}} ref={bannerRef} data-testid={"overview-nav-container"} className="w-100 d-flex flex-column align-items-center justify-content-center sticky-top p-0">
+            <Container style={{ maxWidth:"none", borderBottom: '1px #D6DDE6 solid'}} ref={bannerRef} data-testid={"overview-nav-container"} className="w-100 d-flex flex-column align-items-center justify-content-center sticky-top p-0">
```

Comment:
As we get more finalized designs, I think it's time to start moving this sort of stuff to css. #D6DDE6 should also not just be some random color, and if it's in our sass file we can refer to it as $gray-100 or whatever it ends up being (still working on the bootstrap vars piece of work)

Reply:
Yeah I purposely left that out because I know you were working on the color variables and I didn't want to step on your toes there.